cmd/gossamer/config.go

```diff
@@ -408,6 +410,13 @@ func setDotGlobalConfig(ctx *cli.Context, tomlCfg *ctoml.Config, cfg *dot.Global
 	// check --name flag and update node configuration
 	if name := ctx.GlobalString(NameFlag.Name); name != "" {
 		cfg.Name = name
+	} else {
+		// generate random name
+		entropy, _ := bip39.NewEntropy(128)
+		randomNamesString, _ := bip39.NewMnemonic(entropy)
+		randomNames := strings.Split(randomNamesString, " ")
```

Comment:
Is `len(randomNames) > 1` always?

Reply:
Yes, bip39.NewMnemonic always returns 12 words when provided with a 128 bit entropy.